language/move-lang/tests/move_check/typing/bind_wrong_arity.exp

```diff
@@ -0,0 +1,44 @@
+error: 
+
+   ┌── tests/move_check/typing/bind_wrong_arity.move:5:13 ───
+   │
+ 5 │         let x: () = ();
+   │             ^ Invalid list-binding. The list-binding expected 1 argument(s) but got 0
```

Comment:
List binding is confusing to me. Is there a better name that could be chosen for this?

Reply:
I don't use the term tuple anywhere so I was kind of stuck. I refer to them as expression-lists on the expression errors